style.css

```diff
@@ -165,7 +165,24 @@ nav ul li {
   }
 }
 
-/* end of pop */
+.form-validation {
+  display: flex;
+  align-items: baseline;
+}
+
+.success {
+  color: green;
+}
+
+#error-msg {
```

Comment:
- [ ] Please don't use Ids as selectors instead of ids use classes as you have benefits like multiple use of classes.